x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/forecasts_table/forecasts_table.js

```diff
@@ -21,12 +21,12 @@ import {
 
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { withKibana } from '@kbn/kibana-react-plugin/public';
+import { withKibana, context } from '@kbn/kibana-react-plugin/public';
```

Comment:
Looks like the same can be done here, we can remove `withKibana` as we now use `context`.

Reply:
Updated in 70a4278a495651f8acb2d17f57cf79570ac71c26